docs/representers/normalization.md

```diff
@@ -144,3 +144,45 @@ public static class Fake
     }
 }
 ```
+
+## Normalize the order where insignificant
+
+In some cases the order or code does not matter. To prevent the same code in different order to create different representations it might be useful to sort it. Usually the items to sort are functions or declarations. It might be tricky to find the metric(s) to sort by and also tricky to implement. One metric could be the how many AST node children the node contains, another the name of the type the first child node.
+
+This example sorts by some kind of function length:
+
+### Source code
+
+```csharp
+public static class Fake
+{
+    public static string Test2()
+    {
+        int a = "Test2";
```

Comment:
Would it make sense to also have multiple identifiers here (`a`, `b`, `c`) that are independent and could thus also be sorted?

Reply:
That would probably be ideal but definitely hard to implement. What would you sort them by? Can't use the name. Type would be possible but if they are all of the same type? Sort by context of the first usage maybe?

Reply:
@ErikSchierboom Can you enhance the example? I'm not that familiar with C# to create a good example.

Reply:
Maybe for now let's ignore it.

Reply:
ok